app/src/androidTest/java/ch/epfl/balelecbud/utility/storage/CachedStorageTest.java

```diff
@@ -91,4 +84,28 @@ public void flush() {
 
     }
 
+    private static class MockStorages implements Storage {
```

Comment:
Typo in class name
```suggestion
    private static class MockStorage implements Storage {
```

Reply:
I get that it is conflicting with an already existing class name, but try to find a more pertinent one.